tests/tensor_search/test_add_documents_use_existing_tensors.py

```diff
@@ -85,6 +86,100 @@ def test_use_existing_tensors_non_existing(self):
             document_id="123", show_vectors=True)
         self.assertEqual(use_existing_tensors_doc, regular_doc)
 
+        tensor_search.delete_index(config=self.config, index_name=self.index_name_1)
```

Comment:
if you delete the index, then you don't really overwrite the previous document

Reply:
done